README.md

```diff
@@ -3,37 +3,32 @@
 * Your machine runs Linux !
 * [git](https://gist.github.com/derhuerst/1b15ff4652a867391f03#file-linux-md) is installed
 * [Docker](https://docs.docker.com/install/ is installed)
-* docker-compose is installed (using docker_script.sh can be a workaround if you do not have docker-compose
-* 8000, 8080 are free ports on your machine
+* docker-compose is installed 
+* 7800, 8000, 8080 are free ports on your machine
 
 ## Quick setup
-### You do not have acces to Schweizmobil data
-* Clone the repository
-`git clone https://github.com/camptocamp/vector-tiles-demo.git` in a terminal
-* launch the script which will run the app in a docker container
-```
-$ cd vector-tiles-demo
-$ docker-compose up
-```
-in a terminal
-### You have access to Schweizmobil data
 
-* Requires [tippecanoe](https://github.com/mapbox/tippecanoe) to be installed
-* Requires [ogr2ogr](http://www.sarasafavi.com/installing-gdalogr-on-ubuntu.html) to be installed
+### You already have SwitzerlandMobility data
+
 * Clone the repository :
 `git clone https://github.com/camptocamp/vector-tiles-demo.git` in a terminal
 * move the data : geometryroute_2018-02-16.tgz into the vector-tiles-demo/data directory (create it if it does not exist)
 * download GeoJSON of SBB POI from this page `https://data.sbb.ch/explore/dataset/station-didok/export/` and put it in the vector-tiles-demo/data directory
-* launch the script which will create the vector tiles from the data, and then launch the app in a docker container:
-* launch a cors enabled server on port 7800 at vector-tile-demo/tiles/sprites
+* go to the root directory of the project and type the following command
 ```
-$ cd vector-tiles-demo
-$ docker build -t nhofer/tile-demo .
-$ ./prepare_data.sh
-$ cd tiles
-$ npm start
+docker-compose -f prepare_data.yml build conversion && \
+docker-compose -f prepare_data.yml build dirty-reprojection && \
+docker-compose -f prepare_data.yml build generate-tiles && \
+docker-compose -f prepare_data.yml run conversion && \
+docker-compose -f prepare_data.yml run dirty-reprojection && \
+docker-compose -f prepare_data.yml run generate-tiles && \
+docker-compose -f docker-compose-prod.yml up --build
 ```
 
+### You do not have SwitzerlandMobility data
+
+* contact Kalaboum, if you are authorized to have acces to switzerland mobility data, he will provide you with a one-line command which starts the app
```

Comment:
Once the one-step build commands are final, you could also publish them on Confluence (which is not accessible publicly) and just add a link here.